packages/kokoas-server/src/handleRequest/reqSendContractDirectV2/getContractDataV2.ts

```diff
@@ -222,8 +231,8 @@ export const getContractDataV2 = async (
     payMethod: payMethod.value as '持参' | '集金' | '振込',
 
     /* 会社情報 */
-    companyAddress: companyAddress.value,
-    companyName: companyName.value,
+    companyAddress: `${companyAddress.value} ${storeName.value}`,
```

Comment:
```suggestion
    companyAddress: `${companyAddress.value} ${storeName.value}`,
    companyAddress2: `${companyAddress.value} ハウスドゥ ${storeName.value}`,
```

1案なので、修正はこの通りである必要はないです。実装方法はお任せします。
契約書上部と下部で住所の表記が異なるとのことです。

Reply:
ありがとうございます。そのまま適用しました。